var/spack/repos/builtin/packages/py-configparser/package.py

```diff
@@ -29,16 +29,13 @@ class PyConfigparser(PythonPackage):
     """This library brings the updated configparser from Python 3.5 to
     Python 2.6-3.5."""
 
-    homepage = "https://pypi.python.org/pypi/configparser"
-    url      = "https://pypi.python.org/packages/source/c/configparser/configparser-3.5.0.tar.gz"
+    homepage = "https://docs.python.org/3/library/configparser.html"
+    url      = "https://pypi.io/packages/source/c/configparser/configparser-3.5.0.tar.gz"
 
-    version('3.5.0', 'cfdd915a5b7a6c09917a64a573140538',
-            url="https://pypi.python.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/configparser-3.5.0.tar.gz")
+    version('3.5.0', 'cfdd915a5b7a6c09917a64a573140538')
 
-    depends_on('python@2.6:2.8,3.4:')
```

Comment:
To be honest, this is such an intricate bug that I don't even remember anymore. It might work, it might not. From my experience, if you don't specify a version of Python (use the default) and the dependency isn't added for that version of Python, things crash. So it might work. But I can't remember what happens if you change the default in `packages.yaml`. Long story short: Python support is pretty broken for backports, and for Python 3.